app/models/uploaded_file.rb

```diff
@@ -0,0 +1,14 @@
+class UploadedFile < ApplicationRecord
+
+  belongs_to :membership_application
+
+  has_attached_file :actual_file
+
+  validates_attachment :actual_file, presence: true,
+                       content_type: { content_type: ['image/jpeg', 'image/gif', 'image/png',
+                                                     'text/plain',
+                                                     'text/rtf',
+                                                     'application/pdf',
+                                                     'application/msword'],
+                                       message: 'Sorry, this is not a file type you can upload.'}
```

Comment:
Argh -- I'd seen that ugly MIME type and was hoping it wasn't really needed.  I'm glad to learn of your experience: that _is_ the MIME type for the .docx document type.